collaboration-and-sharing/public-shareable-links.md

```diff
@@ -0,0 +1,31 @@
+# Create public shareable links
+
+{% hint style="info" %}
+**Enterprise plan only**
+This feature is only available in custom enterprise pricing plans.
+{% endhint %}
+
+When an asset or media collection needs to be shared externally with someone outside your ImageKit environment, a public shareable link can be created for it. Public links allow limited, read-only access to your assets without the need to register for an ImageKit account. This includes the ability to download multiple assets in a ZIP file. A link can be created for a file, folder, or media collection. Configurable options for a public link include password protection, a validity timeframe, and whether to expose [draft assets](../media-library/overview/draft-assets.md) or not. If an expiration date is set for a public link, it will be automatically deleted after expiration.
+
+For restricted media library users, the minimum permission level needed to create, view, and edit public links for an asset or media collection is 'Can manage'. Learn more about permissions and access sharing for restricted users [here](../collaboration-and-sharing/README.md#understanding-permissions-on-assets-and-media-collections).
+
+{% hint style="info" %}
+**Limits**
+For a single asset or media collection, a maximum of 5 public links can be created. Across all assets and media collections in an account, a maximum of 100 public links can be created. This limit is configurable.
+{% endhint %}
+
+You can create a public link for any asset or media collection in your media library.
+
+![Right-click menu location for public links](<../.gitbook/assets/right-click-menu-public-links.png>)
+
+![Form for creating a new public link](<../.gitbook/assets/new-public-link-form.png>)
+
+On the media library settings page, public links for all assets and media collections in your account can be seen.
+
+![Table displaying all public links in an account](<../.gitbook/assets/all-public-links-table-settings-page.png>)
+
+## Downloads and public links
+
+Downloads triggered via a public link will appear on your downloads page with the 'Via public link' field set to 'Yes'.
```

Comment:
Public users can see downloads in their session.

Account users can see downloads triggered via public links.

Reply:
split into two sentences